agent/apps/ai_guardrails/app_code/serializers.py

```
@@ -0,0 +1,93 @@
+
+class GuardrailPolicySerializer(BaseModelSerializer):
+    """
+    سریالایزر سیاست گاردریل
+    """
+
+    class Meta:
+        model = GuardrailPolicy
+        fields = [
+            'id', 'name', 'description', 'is_active', 'enforcement_mode',
+            'applies_to', 'priority', 'conditions', 'created_at', 'updated_at', 'created_by'
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)